tools/shell/embedded_shell.cpp

```diff
@@ -37,15 +37,17 @@ struct ShellCommand {
 
 const char* TAB = "    ";
 
-const std::array<const char*, 82> keywordList = {"CALL", "CREATE", "DELETE", "DETACH", "EXISTS",
+const std::array<const char*, 104> keywordList = {"CALL", "CREATE", "DELETE", "DETACH", "EXISTS",
```

Comment:
LGTM.

Can u also open an issue about synchronizing our Lexer rule with keyword highlighting in the shell. Ideally, we should maintain our keywords in one place that use it as the source of truth to generate our Lexer and keywords.